docs/docs/dev_docs/getting_started/cli.md

```diff
@@ -0,0 +1,230 @@
+
+- `--args` - The address for which we want to retrieve the balance.
+- `--contract-abi` - The abi of the contract we are calling.
+- `--contract-address` - The address of the deployed contract
+
+As you can see from the result, this address has a balance of 1000000, as expected.
+
+## Sending a Transaction
+
+We can now send a transaction to the network. We will transfer funds from the owner of the initial minted tokens to our other account. For this we will use the `send` command, which expects as arguments the quantity of tokens to be transferred, the sender's address, and the recipient's address. Make sure to replace all addresses in this command with the ones for your run.
+
+```
+% aztec-cli send transfer \
+  --args 543 0x2337f1d5cfa6c03796db5539b0b2d5a57e9aed42665df2e0907f66820cb6eebe 0x175310d40cd3412477db1c2a2188efd586b63d6830115fbb46c592a6303dbf6c \
+  --contract-abi PrivateTokenContractAbi \
+  --contract-address 0x1ae8eea0dc265fb7f160dae62cc8912686d8a9ed78e821fbdd8bcedc54c06d0f \
+  --private-key 0x2aba9e7de7075deee3e3f4ad1e47749f985f0f72543ed91063cc97a40d851f1e
+
+Transaction has been mined
+Transaction hash: 15c5a8e58d5f895c7e3017a706efbad693635e01f67345fa60a64a340d83c78c
+Status: mined
+Block number: 5
+Block hash: 163697608599543b2bee9652f543938683e4cdd0f94ac506e5764d8b908d43d4
+```
+
+We called the `transfer` function of the contract and provided these arguments:
+
+- `--args` - The list of arguments to the function call.
+- `--contract-abi` - The abi of the contract to call.
+- `--contract-address` - The deployed address of the contract to call.
+
+The command output tells us the details of the transaction such as its hash and status. We can use this hash to query the receipt of the transaction at a later time:
+
+```
+% aztec-cli get-tx-receipt 15c5a8e58d5f895c7e3017a706efbad693635e01f67345fa60a64a340d83c78c
+
+Transaction receipt:
+{
+  "txHash": "15c5a8e58d5f895c7e3017a706efbad693635e01f67345fa60a64a340d83c78c",
+  "status": "mined",
+  "error": "",
+  "blockHash": "163697608599543b2bee9652f543938683e4cdd0f94ac506e5764d8b908d43d4",
+  "blockNumber": 5,
+  "origin": "0x2337f1d5cfa6c03796db5539b0b2d5a57e9aed42665df2e0907f66820cb6eebe"
+}
+```
+
+Let's now call `getBalance()` on each of our accounts and we should see updated values:
+
+```
+% aztec-cli call getBalance -a 0x2337f1d5cfa6c03796db5539b0b2d5a57e9aed42665df2e0907f66820cb6eebe -c PrivateTokenContractAbi -ca 0x1ae8eea0dc265fb7f160dae62cc8912686d8a9ed78e821fbdd8bcedc54c06d0f
+
+View result:  [
+  "{\"type\":\"bigint\",\"data\":\"999457\"}"
+]
+
+% aztec-cli call getBalance -a 0x175310d40cd3412477db1c2a2188efd586b63d6830115fbb46c592a6303dbf6c -c PrivateTokenContractAbi -ca 0x1ae8eea0dc265fb7f160dae62cc8912686d8a9ed78e821fbdd8bcedc54c06d0f
+
+View result:  [
+  "{\"type\":\"bigint\",\"data\":\"543\"}"
+]
+```
+
+## Logs
+
+Finally, we can use the CLI's `get-logs` command to retrieve unencrypted logs emitted by the contract:
+
+```
+% aztec-cli get-logs 5 1
```

Comment:
Possibly worth opening an issue to use named arguments here. `--from-block-num 5 --num-blocks 1`.

Reply:
It was faster to open a PR https://github.com/AztecProtocol/aztec-packages/pull/1559

Reply:
Haha amazing!